plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java

```diff
@@ -67,20 +63,11 @@ private static ClickHouseContainer createContainer(DockerImageName image)
             @Override
             public String getDriverClassName()
             {
-                return getClickhouseDriverClassName(image);
+                return CLICKHOUSE_LATEST_DRIVER_CLASS_NAME;
             }
```

Comment:
Do we need to override this `getDriverClassName` - won't `ClickHouseContainer` maps to the latest one.